packages/gestalt/src/Table/Table.test.js

```diff
@@ -1,7 +1,7 @@
 // @flow strict
 import React from 'react';
 import renderer from 'react-test-renderer';
-import Table from './Table.js';
+import Table from './index.js';
```

Comment:
@simkessy we should avoid `index.js` files when possible. It's hard to know what's in those files + they don't autocomplete well

Reply:
I think my comment below also applies regarding the use of Barrels. It's a practice I've seen quite often, i'm not entirely clear why we'd deviate. Regarding the issue of confusion, I think if you're looking at the `/Table/index.js`, I don't find it confusing that this file is the entry point to the Table component. Same as how the `gestalt/src/index.js` file is the entry point to Gestalt. 

If someone is not familiar with Barrels, it would be a good learning opportunity.